build/commands/scripts/commands.js

```diff
@@ -358,6 +359,10 @@ program
   .option('--vmodule [modules]', 'verbose log from specific modules')
   .option('--filter <filter>', 'set test filter')
   .option('--no_gn_gen', 'Use args.gn as default values')
+  .option(
+    '--since_commit [targetCommit]',
```

Comment:
as mentioned above specifying a target commit is generally not super helpful, using `--base` would be better and allow a branch/tag/commit ref, but I this this option and the new command need a console warning that they are experimental for now

Reply:
used stderror and added -q flag to allow suppression of this message

Reply:
> and yes it supports branch/tag/commit ref too

I think using the same option name as presubmit would still be good here for consistency